external/fv3fit/fv3fit/tfdataset.py

```diff
@@ -13,6 +13,13 @@ def apply_to_mapping(
     return {name: tensor_func(tensor) for name, tensor in data.items()}
 
 
+@curry
+def apply_to_tuple(
```

Comment:
where is this used? Doesn't seem specific to to `tf.data` since it's more of a general purpose higher-order-function.